jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java

```diff
@@ -23,10 +23,10 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
+
+import org.assertj.core.api.InstanceOfAssertFactories;
 import org.eclipse.jkube.generator.api.GeneratorContext;
-import org.eclipse.jkube.kit.common.JavaProject;
-import org.eclipse.jkube.kit.common.KitLogger;
-import org.eclipse.jkube.kit.common.Plugin;
+import org.eclipse.jkube.kit.common.*;
```

Comment:
could you please remove these wildcard imports?

Reply:
Sorry for not being clear, I was referring to getting rid of wildcard style imports (`.*`) and keeping single imports, now build is failing due to missing imports

```
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project jkube-kit-generator-webapp: Compilation failure: Compilation failure: 
[ERROR] /home/jenkins/agent/workspace/Build_Dependencies_PR-3302/jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java:[131,83] cannot find symbol
[ERROR]   symbol:   class AssemblyFile
[ERROR]   location: class org.eclipse.jkube.generator.webapp.WebAppGeneratorTest.CustomizeAssembly
[ERROR] /home/jenkins/agent/workspace/Build_Dependencies_PR-3302/jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java:[177,83] cannot find symbol
[ERROR]   symbol:   class AssemblyFile
[ERROR]   location: class org.eclipse.jkube.generator.webapp.WebAppGeneratorTest.CustomizeAssembly
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project jkube-kit-generator-webapp: Compilation failure
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:375)
```

Reply:
Sorry @rohanKanojia   I didn't Cross check with the imports previously so i made changes  and   now Cross Checked with all the imports i dont see any error now

Reply:
@ShivamChavan01 : okay, it's better to verify that the build is passing locally before pushing changes.

Reply:
![image](https://github.com/user-attachments/assets/b42acf3f-8f22-43c5-b432-6e443828db26)

 All Done